mirgecom/boundary.py

```diff
@@ -1277,22 +1475,3 @@ def viscous_wall_flux(self, discr, btag, gas_model, state_minus,
                              grad_t=grad_t_plus)
 
         return f_ext@normal
-
-    def adiabatic_slip_grad_av(self, discr, btag, grad_av_minus, **kwargs):
-        """Get the exterior grad(Q) on the boundary."""
-        # Grab some boundary-relevant data
-        dim, = grad_av_minus.mass.shape
-        actx = grad_av_minus.mass[0].array_context
-        nhat = actx.thaw(discr.norm(btag))
-
-        # Subtract 2*wall-normal component of q
-        # to enforce q=0 on the wall
-        s_mom_normcomp = np.outer(nhat,
-                                  np.dot(grad_av_minus.momentum, nhat))
-        s_mom_flux = grad_av_minus.momentum - 2*s_mom_normcomp
-
-        # flip components to set a neumann condition
-        return make_conserved(dim, mass=-grad_av_minus.mass,
-                              energy=-grad_av_minus.energy,
-                              momentum=-s_mom_flux,
-                              species_mass=-grad_av_minus.species_mass)
```

Comment:
Not sure about this.  I think maybe we should leave AV alone.   We probably need to call this function if @anderson2981 is using AV with this boundary.